assets/js/base/components/tabs/index.tsx

```diff
@@ -10,6 +10,22 @@ import { useTabState, Tab, TabList, TabPanel } from 'reakit/Tab';
  */
 import './style.scss';
 
+export interface TabsProps {
+	className?: string;
+	onSelect?: ( tabName: string ) => void;
+	tabs: Array< {
+		name: string;
+		title: string;
+		content: JSX.Element;
+		ariaLabel?: string;
+	} >;
+	activeClass?: string;
+	initialTabName?: string | undefined;
```

Comment:
@tarunvijwani, the `initialTabName` is marked as optional (might be undefined). So, we don't need to specify that it can be undefined.

```suggestion
	initialTabName?: string;
```

Reply:
This is not true, because we have `exactOptionalPropertyTypes` enabled https://github.com/woocommerce/woocommerce-blocks/blob/d9f0aa8b8e26bbf50f109ac1f845b22ac8be208b/tsconfig.base.json#L10 we do need to define it.